orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/OperationsController.groovy

```diff
@@ -204,6 +206,53 @@ class OperationsController {
     }
   }
 
+  private void addStageAuthorizedRoles(def request, Map pipeline) {
+
+    if (fiatStatus && fiatStatus.isEnabled()) {
+      def applicationName = pipeline.application
+      if (applicationName) {
+        Optional<Application> optApplication = getApplication(applicationName)
+        if (optApplication.isPresent()) {
+          Application application = optApplication.get();
```

Comment:
nit: Could be simplified to `getApplication(applicationName).ifPresent(application -> { /* ... */ });`

Reply:
done